start-all.sh

```diff
@@ -591,7 +591,17 @@ if [[ $DOCKER_PARAM = *"--net=host"* ]]; then
 		echo "Port mapping is not supported with host network, remove the -l flag from the command line"
 		exit 1
 	fi
-	HOST_NETWORK=1
+	HOST_NETWORK=host
+	GRAFANA_ADDRESS="-G localhost:3000"
+elif [[ ! $DOCKER_PARAM = *"--net"* ]]; then
+    HOST_NETWORK="monitor-net"
+    if [ "$STACK_ID" != "" ]; then
+        HOST_NETWORK="$HOST_NETWORK$STACK_ID"
+    fi
+    if ! docker network inspect $HOST_NETWORK >/dev/null 2>&1; then
+      docker network create $HOST_NETWORK
```

Comment:
I wonder - do we now need root permissions, which we did not so far, or maybe it doesn't matter as Docker runs as root anyway?

Reply:
<img width="729" height="553" alt="image" src="https://github.com/user-attachments/assets/efd5005f-9e5e-4885-b4c6-3906226ec540" />